include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -869,20 +881,47 @@ struct SPhysicalDeviceLimits
 
         if (minImportedHostPointerAlignment < _rhs.minImportedHostPointerAlignment) return false;
 
+        if (shaderBufferFloat32AtomicAdd && !_rhs.shaderBufferFloat32AtomicAdd) return false;
+        if (shaderBufferFloat64Atomics && !_rhs.shaderBufferFloat64Atomics) return false;
+        if (shaderBufferFloat64AtomicAdd && !_rhs.shaderBufferFloat64AtomicAdd) return false;
+        if (shaderSharedFloat32AtomicAdd && !_rhs.shaderSharedFloat32AtomicAdd) return false;
+        if (shaderSharedFloat64Atomics && !_rhs.shaderSharedFloat64Atomics) return false;
+        if (shaderSharedFloat64AtomicAdd && !_rhs.shaderSharedFloat64AtomicAdd) return false;
+        if (shaderImageFloat32AtomicAdd && !_rhs.shaderImageFloat32AtomicAdd) return false;
+        if (sparseImageFloat32Atomics && !_rhs.sparseImageFloat32Atomics) return false;
+        if (sparseImageFloat32AtomicAdd && !_rhs.sparseImageFloat32AtomicAdd) return false;
+
+        if (robustStorageBufferAccessSizeAlignment < _rhs.robustStorageBufferAccessSizeAlignment) return false;
+        if (robustUniformBufferAccessSizeAlignment < _rhs.robustUniformBufferAccessSizeAlignment) return false;
+
+        if (shaderTrinaryMinmax && !_rhs.shaderTrinaryMinmax) return false;
+        if (shaderExplicitVertexParameter && !_rhs.shaderExplicitVertexParameter) return false;
+        if (gpuShaderHalfFloatAMD && !_rhs.gpuShaderHalfFloatAMD) return false;
+        if (shaderImageLoadStoreLod && !_rhs.shaderImageLoadStoreLod) return false;
+        if (displayTiming && !_rhs.displayTiming) return false;
```

Comment:
have spaces after members that belong to separate exxtensions (see how they're laid out in `SPhyscalDeviceLimits` above)